scspell/_util.py

```diff
@@ -60,8 +60,20 @@ def open_with_encoding(filename, encoding=None, mode='r'):
 
 def detect_encoding(filename):
     """Return file encoding."""
+
+    # Thanks again stackoverflow for python 2 v 3,
+    # http://stackoverflow.com/questions/15032108/pythons-open-throws-different-errors-for-file-not-found-how-to-handle-b
+    fnotfound = getattr(__builtins__, 'FileNotFoundError', IOError)
```

Comment:
My intention was for this to handle only file-not-found, and continue to propagate e.g. permission errors up the stack as before.

Though now that I think I understand that getattr() line a little better, I see that's not what it's doing.